diesel/src/query_source/joins.rs

```diff
@@ -154,8 +154,8 @@ where
 // }
 
 /// Indicates that two tables can be used together in a JOIN clause.
-/// Implementations of this trait will be generated for you automatically by
-/// the [association annotations](../associations/index.html) from codegen.
+/// Implementations of this trait will be generated for you when using
+/// the [joinable!](../macro.joinable.html) macro.
```

Comment:
We should mention here that calls to that macro are generated by `infer_schema!` and `print-schema` if the database relations exists in SQL (`REFERENCES`)

Also we should mention here the [`allow_tables_to_appear_in_same_query!`](http://docs.diesel.rs/diesel/macro.allow_tables_to_appear_in_same_query.html) macro that enables multi table joins.